tools/projmgr/src/ProjMgrWorker.cpp

```diff
@@ -2169,13 +2171,15 @@ bool ProjMgrWorker::ProcessSequencesRelatives(ContextItem& context, BuildType& b
     return false;
   }
   for (auto& misc : build.misc) {
-    if (!ProcessSequencesRelatives(context, misc.as)    ||
-        !ProcessSequencesRelatives(context, misc.as)    ||
-        !ProcessSequencesRelatives(context, misc.c)     ||
-        !ProcessSequencesRelatives(context, misc.cpp)   ||
-        !ProcessSequencesRelatives(context, misc.c_cpp) ||
-        !ProcessSequencesRelatives(context, misc.lib)   ||
-        !ProcessSequencesRelatives(context, misc.link)) {
+    if (!ProcessSequencesRelatives(context, misc.as)     ||
+        !ProcessSequencesRelatives(context, misc.as)     ||
```

Comment:
Can you please remove one of the `ProcessSequencesRelatives(context, misc.as)`? Seems like this is a mistake.

Reply:
Sorry, it's my fault when rebasing the main branch. THanks for the careful catch!